src/styles/forms.scss

```diff
@@ -1,10 +1,136 @@
+@import '@/styles/colors.scss';
+
 .select {
     background-color: white;
     border: 1px solid #ced4da;
     border-radius: .25rem;
     box-sizing: border-box;
     color: #495057;
-
     font-size: 1rem;
     padding: 6px 4px;
 }
+
+// Organization -----------------------------------
+
+.legend {
+    color: $ocean-blue;
+    font-size: 18px;
+}
+
+.fieldset {
+    border-bottom: none;
+    border-left: none;
+    border-right: none;
+    border-color: rgba(255, 255, 255, 0.3);
+    border-width: 2px;
+}
+
+.input-tooltip {
+    color: mediumvioletred;
+    font-size: 15px;
+    margin-left: 8px;
+}
+
+.text-label {
+    color: lighten(black, 10);
+    display: inline-block;
+    font-size: 15px;
+    font-weight: 700;
+    margin: 0 0 7px 0;
+    text-align: right;
+}
+
+// Checkbox ---------------------------------------
+
+.checkbox-label {
+    color: lighten(black, 10);
+    display: inline-block;
+    font-size: 15px;
+    font-weight: 700;
+    margin: 0 0 0 0;
+}
+
+.checkbox-input-container {
+    padding: 10px 0 10px 5px;
+}
+
+.checkbox  {
+    clip: rect(0,0,0,0);
+    position: absolute;
+}
+
+.checkbox + label::before {
+    border-radius: 2px;
+    color: #ced4da;
+    content: '\f0c8';
+    display: inline-block;
+    font-family: "Font Awesome 5 Free";
```

Comment:
I'm not crazy about the way the checkboxes look. Just having a gray square doesn't really look like a checkbox for me. 
I'd be in support of either the default way checkboxes look or some kind of rectangle with a black border and a white center (like the way they look on google forms).